.github/workflows/ci.yaml

```diff
@@ -102,11 +102,10 @@ jobs:
           docker compose run --rm \
             --volume $DATADIR:/in \
             cumulus-etl \
+            nlp \
```

Comment:
The fact that this change was needed here but not before, tells me we are regression testing the public release instead of the current branch. We have code to build the branch's docker image... not sure why it's not being used here. I can dig into it later, but just an FYI - it's on my todo list.